src/error-table.rkt

```diff
@@ -0,0 +1,49 @@
+#lang racket
+
+(require "points.rkt" "syntax/types.rkt" "core/localize.rkt" "timeline.rkt")
+
+(provide group-errors)
+
+(define (zip l1 l2) (map list l1 l2))
```

Comment:
I don't really see the point of making this a helper method. Instead of iterating over `(zip a b)` the Racket `for` loop can just iterate over multiple things in parallel (see below where you use `zip`).